runtime/containers.go

```diff
@@ -80,6 +81,13 @@ func (t *ContainerTarget) ToolsDir(ctx context.Context) string {
 	return containerDefaultToolsDir
 }
 
+func (t *ContainerTarget) ToolOutputSharedDir(ctx context.Context) string {
+	err := narwhal.MkdirAll(ctx, narwhal.DockerClient(), t.Container.Id, containerDefaultToolOutputSharedDir)
+	if err != nil {
+		return ""
+	}
```

Comment:
Do we need to log this error, or is it an expected case?

Reply:
Good catch, thanks